docs/Oracle Jobs/migration-v1-v2.md

```diff
@@ -140,7 +140,7 @@ observationSource   = """
                       "expiration": $(decode_log.cancelExpiration),
                       "data": $(encode_data)
                   }>]
-    submit       [type=ethtx to="$(jobSpec.contractAddress)" data="$(encode_tx)"]
+    submit       [type=ethtx to="0x613a38AC1659769640aaE063C651F48E0250454C" data="$(encode_tx)"]
 
     decode_log -> decode_cbor -> fetch -> parse -> encode_data -> encode_tx -> submit
```

Comment:
Can this be `submit_tx` to be consistent with the other examples, or does it affect how this job functions?

Reply:
doesnt affect the function - will do